### PR TITLE
MM-40566 Remove collapse from websocket sync

### PR DESCRIPTION
**Pull request by Safouene1**
#### Summary
[OLD PR before Monorepo ](https://github.com/mattermost/mattermost-webapp/pull/12186)
Stop Websocket from changing category.collapse state.
Keep Websocket syncing , just not change the state only for Collapse/Expand state.
**More details  in the Issue linked below.** 

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/22044

 JIRA ticket : https://mattermost.atlassian.net/browse/MM-40566

#### Release Note

```release-note
remove websocket state effects for the collapse/expand state of categories
```

### CODE REVIEW

**Comment by hmhealey** [comment]
Thanks for moving this over! This looks good to me, but it looks like the linter has some complaints. Mind fixing those?

**Comment by Safouene1** [comment]
> Thanks for moving this over! This looks good to me, but it looks like the linter has some complaints. Mind fixing those?
Sorry about those , I forgot to turn on the linter for the monorepo , I pushed the changes

**Review by hmhealey — APPROVED** [review]

**Comment by hmhealey** [comment]
Looks good to me. The remaining CI failures are just due to the code that prepares test servers not being able to run on community PRs right now

**Comment by Safouene1** [comment]
Hey , Are the failing E2E tests in relation to my code? Is there anything I can help with?

**Review by stevemudie — APPROVED** [review]
The failed E2E tests don't appear to be related to your work @Safouene1 
This is all set. Thank you!

**Comment by hmhealey** [comment]
I'm going to give this another update to see if that fixes CI. I have no idea why any of that would be failing based on the changes you've made

/update-branch

**Review by hmhealey — CHANGES_REQUESTED** [review]
@Safouene1 I didn't realize the type check is actually failing because of these changes. Would you mind fixing that?

It looks like some Playbooks code is using `fetchMyCategories`, so we could make `isWebSocket` default to false to fix it

**Comment by Safouene1** [comment]
> It looks like some Playbooks code is using `fetchMyCategories`, so we could make `isWebSocket` default to false to fix it

Thank god CI exist  , Yes I will update it this week.

**Comment by Safouene1** [comment]
Should work fine now , could someone run the e2e testa

File: webapp/playbooks/src/components/backstage/playbook_editor/playbook_editor.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const PlaybookEditor = () => {
 
         dispatch(selectTeam(teamId));
         dispatch(fetchMyChannelsAndMembersREST(teamId));
-        dispatch(fetchMyCategories(teamId));
+        dispatch(fetchMyCategories(teamId,false));
```
Comment (hmhealey) [inline webapp/playbooks/src/components/backstage/playbook_editor/playbook_editor.tsx:69]:
```suggestion
        dispatch(fetchMyCategories(teamId, false));
```

**Review by hmhealey — CHANGES_REQUESTED** [review]

**Comment by hmhealey** [comment]
Thanks for the quick update. I'll kick those tests off again. It's annoying right now that the CI needs one of us to approve it before it tells you simple stuff like that :/

**Comment by Safouene1** [comment]
> It's annoying right now that the CI needs one of us to approve it before it tells you simple stuff like that :/

 I agree that it would be helpful to have at least the linting and i18n tests launch automatically without requiring manual approval. That would save a lot of time and effort in the long run.

**Comment by hmhealey** [comment]
Thanks again for that change! I definitely appreciate this one since I'm the one that introduced the bug, but I never got around to fixing it myself

**Pull request merged**